src/proc_image_processing/cpu/algorithm/general_function.h

```diff
@@ -0,0 +1,130 @@
+/// \author	Pierluc Bédard <pierlucbed@gmail.com>
+/// \author	Jérémie St-Jules Prévôt <jeremie.st.jules.prevost@gmail.com>
```

Comment:
Remove the lines of the authors' name? Since we can trace back the authors from source control.